### DISCUSSION TITLE
v1.0.3.8: sata_remap supported?

### DISCUSSION THREAD

**Discussion by Zitt** [PeterSuh-Q3/tinycore-redpill#39] [Q&A] [answered]
Peter,
I've had some success booting your redpill / mshell on a Asus x299 board with onboard sata controller.

I've been trying to implement sata_remap as outline in this post:
https://xpenology.com/forum/topic/69296-how-to-remap-sata-ports-arpl-so-that-your-drives-are-shown-in-the-proper-order-in-dsm/

Is this supported in mshell? On my custom nas solution; the ports are misnumbered:
![x299Nas](https://github.com/PeterSuh-Q3/tinycore-redpill/assets/13250637/9704d517-0e8c-4afc-b2b6-1867a03f9a4a)

I tried the following options in user_config.json with no success:

  "extra_cmdline": {
    "pid": "0x6387",
    "vid": "0x058f",
    "sn": <redacted>
    "mac1": <redacted>
    "netif_num": "3",
    "SataPortMap": "8",
    "DiskIdxMap": "00",
    "mac2": <redacted>
    "mac3": <redacted>
    "sata_remap": "0\\>1:1\\>0:3\\>2:6\\>3"
  }

Any thoughts?

**Comment by PeterSuh-Q3** [comment, marked as answer]
Fix like this...
https://xpenology.com/forum/topic/61839-tinycore-redpill-loader-build-support-tool-m-shell/?do=findComment&comment=465634
